net/mosquitto/Portfile

```diff
@@ -19,17 +20,17 @@ long_description    \
     Mosquito provides an open-source MQTT v3.1.1 broker (i.e., server) and \
     both  C and C++ client libraries.
 
-
 homepage            https://mosquitto.org
 master_sites        http://mosquitto.org/files/source/
 
-checksums           rmd160 e3066723255dfffbf2f86fc4d37625c866072477 \
-                    sha256 156b1fa731d12baad4b8b22f7b6a8af50ba881fc711b81e9919ec103cf2942d1
+checksums           rmd160  d2438c8c3b2005265def411826961f94c9413b52 \
+                    sha256  7d3b3e245a3b4ec94b05678c8199c806359737949f4cfe0bf936184f6ca89a83 \
+                    size    368961
 
-depends_lib         port:c-ares \
-                    port:tcp_wrappers \
-                    port:libwebsockets \
-                    port:openssl
+depends_lib         path:lib/libcares.dylib:c-ares \
+                    path:lib/libwebsockets.dylib:libwebsockets \
+                    path:lib/libwrap.dylib:tcp_wrappers \
+                    path:lib/libssl.dylib:openssl
```

Comment:
Why do you need a path dependency for all these ports? We only do this when two ports provide the same file, ideally both should be abi-compatible, so that the port will work without rebuilding when you replace one dependency with the other.

Even `openssl` is problematic and the only proper way would be to implement two variants and change the variant to actually switch to `libressl`.